build/specifications/Docker.json

```diff
@@ -8270,6 +8271,238 @@
+        "properties": [
+          {
+            "name": "AddHost",
+            "type": "List<string>",
+            "format": "--add-host {value}",
+            "help": "Add a custom host-to-IP mapping (host:ip)."
+          },
+          {
+            "name": "Allow",
+            "type": "List<string>",
+            "format": "--allow  {value}",
+            "help": "Allow extra privileged entitlement, e.g. network.host, security.insecure."
+          },
+          {
+            "name": "BuildArg",
+            "type": "List<string>",
+            "format": "--build-arg {value}",
+            "help": "Set build-time variables."
+          },
+          {
+            "name": "Builder",
+            "type": "string",
+            "format": "--builder {value}",
+            "help": "Override the configured builder instance."
+          },
+          {
+            "name": "CacheFrom",
+            "type": "List<string>",
+            "format": "--cache-from {value}",
+            "help": "External cache sources (eg. user/app:cache, type=local,src=path/to/dir)."
+          },
+          {
+            "name": "CacheTo",
+            "type": "List<string>",
+            "format": "--cache-to {value}",
+            "help": "Cache export destinations (eg. user/app:cache, type=local,dest=path/to/dir)"
+          },
+          {
+            "name": "CgroupParent",
+            "type": "string",
+            "format": "--cgroup-parent {value}",
+            "help": "Optional parent cgroup for the container."
+          },
+          {
+            "name": "Compress",
+            "type": "bool",
+            "format": "--compress",
+            "help": "Compress the build context using gzip."
+          },
+          {
+            "name": "CpuPeriod",
+            "type": "long?",
+            "format": "--cpu-period {value}",
+            "help": "Limit the CPU CFS (Completely Fair Scheduler) period."
+          },
+          {
+            "name": "CpuQuota",
+            "type": "long?",
+            "format": "--cpu-quota {value}",
+            "help": "Limit the CPU CFS (Completely Fair Scheduler) quota."
+          },
+          {
+            "name": "CpuShares",
+            "type": "long?",
+            "format": "--cpu-shares {value}",
+            "help": "CPU shares (relative weight)."
+          },
+          {
+            "name": "CpusetCpus",
+            "type": "string",
+            "format": "--cpuset-cpus {value}",
+            "help": "CPUs in which to allow execution (0-3, 0,1)."
+          },
+          {
+            "name": "CpusetMems",
+            "type": "string",
+            "format": "--cpuset-mems {value}",
+            "help": "MEMs in which to allow execution (0-3, 0,1)."
+          },
+          {
+            "name": "File",
+            "type": "string",
+            "format": "--file {value}",
+            "help": "Name of the Dockerfile (Default is 'PATH/Dockerfile')."
```

Comment:
I mean things like this. Instead of `'PATH/Dockerfile'` it should be `<c>PATH/Dockerfile</c>`. There's a section on https://nuke.build about that.

Reply:
Done